tools/dockerfile/grpc_artifact_python_manylinux2014_x64/Dockerfile

```diff
@@ -13,18 +13,21 @@
 # limitations under the License.
 
 # Docker file for building gRPC manylinux Python artifacts.
-# Updated: 2021-08-23
+# Updated: 2021-08-27
 
-# Use a tagged version to use Python 3.5
-FROM quay.io/pypa/manylinux2014_x86_64:2021-05-01-28d233a
+# Use a tagged version to use Python 3.10
+FROM quay.io/pypa/manylinux2014_x86_64:2021-08-26-12e5da0
```

Comment:
Lidi, you don't need to use a tag version here. Just `quay.io/pypa/manylinux2014_x86_64` would be better because we can get the latest image once actual python 3.10 is released.

Reply:
Doesn't this open us up to non-hermetic breakage?

Reply:
@veblush I think when a newer upstream Docker image is released, it would be better if we can write a PR for it. We still need to manually build and push the images.

As Richard mentioned, if a certain image version introduce a breakage, it could be hard to debug.

Reply:
+1 to tagging the version and making tests more hermetic that way.